pootle/templates/editor/units/edit.html

```diff
@@ -212,9 +212,15 @@
               {% endblock %}
               <a class="js-toggle-raw"><i class="icon-raw-mode" title="{% trans 'Raw View' %}"></i></a>
             </div>
-            {% if unit.istranslated or unit.isfuzzy %}
-	    {% if unit.change.submitted_by and unit.change.submitted_on %}
             <div class="unit-meta">
+              <span id="js-char-count">
+                {% blocktrans with count=unit_values|map_to_lengths|join:" / " %}
+                  {{ count }} Characters
```

Comment:
Nit: Use `characters` (in lowercase)

Reply:
Also this needs to be able to handle plurals

Reply:
Yes, so the thing with the plurals is that for messages with more than one text area, the string will read `"12 / 13 / 17 / ... characters"` so I wasn't sure how to properly handle plurals on that one.

Reply:
IMHO it is ok to always using "characters" if there is more than a textarea, but if there is only a single textarea I would use the plurals.

Reply:
Or maybe it is better to show the character count for each textarea instead of having the ` / `, but not sure about the impact in the UI. @phlax Any input on this?